examples/PythonProvider/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for quickstartproject project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+import os
+from pathlib import Path
+from azure.appconfiguration.provider import load_provider, AzureAppConfigurationKeyVaultOptions
+from azure.identity import DefaultAzureCredential
+
+ENDPOINT = os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+credential = DefaultAzureCredential()
+keyvault_options = AzureAppConfigurationKeyVaultOptions(credential=credential)
+config = load_provider(endpoint=ENDPOINT, key_vault_options=keyvault_options, credential=credential)
```

Comment:
This is the core part of this example. I think it deserves some comments to explain what it is doing.

I know we are trying to keep the example simple. Can we make it a little bit closer to a real-world app, for example, load keys with a certain prefix or label only.

Reply:
The examples have been updated to trim prefixes.